docs/add-ons/crossplane.md

```diff
@@ -67,4 +68,15 @@ crossplane_jet_aws_provider = {
 }
 ```
 
+_NOTE: Crossplane requires cluster-admin permissions to create and update Kubernetes resources._
+
+Config to deploy [Kubernetes provider](https://github.com/crossplane-contrib/provider-kubernetes)
+```hcl
+# Creates ProviderConfig -> kubernetes-provider
+crossplane_kubernetes_provider = {
+  enable                   = true
+  provider_kubernetes_version     = "main"  # Get the latest version from  https://github.com/crossplane-contrib/provider-jet-aws
```

Comment:
Fix url and pin to specific version.

```suggestion
  provider_kubernetes_version     = "v0.5.0"  # Get the latest version from  https://github.com/crossplane-contrib/provider-kubernetes
```

Reply:
I did initially try with "v0.5.0" but that version doesn't appear to exist in DockerHub: https://hub.docker.com/r/crossplane/provider-kubernetes/tags. The provider was failing to run. I also tried v0.5.0 from the upbound repository, but that was similarly failing to run: https://marketplace.upbound.io/providers/crossplane-contrib/provider-kubernetes/v0.5.0.